### ISSUE TITLE
Thermal Foundation ore problem

### ISSUE THREAD

**Issue by KnowMyName98** [John-Smith-Modded/JSTR-Modded-1.10.x#13]
Hey there everybody, I have been using these textures for the last couple days now and I'm absolutely in love in them, although suffice to say there was one tiny problem that really bugged me.

You see, the ores from Thermal Foundation - the copper, tin, lead, and all that other bullshit were not properly re-textured, save for two: aluminium, and iridium. After taking a quick peak inside the ore folder located inside Thermal Foundation I noticed that these two ores also were the only ones in that folder that weren't capitalized. Seeing how this couldn't be a coincidence, I renamed all the other ores into their non-capitalized counterparts, and presto, everything seems to work now.

So basically, seeing how I don't have editing privileges (or at least I don't think I do) I thought the least I could do would be to report it. For indeed, there is no reason for these beautiful textures to exist, if they aren't properly allocated to the correct blocks :)

**Comment by Oddface** [comment]
Alright, it's fixed. GitHub ignores changes to capitalisation, so you have to delete and re-add the files. Thanks for reporting.
